expo/gbe/cms_plugins.py

```diff
@@ -2,8 +2,25 @@
 from cms.plugin_pool import plugin_pool
 from django.utils.translation import ugettext as _
 from django.template import loader
-from gbe.forms import ContactForm
+from gbe.forms import *
```

Comment:
I would actually like to use explicit imports in general, in this style:
'''
from gbe.forms import (
    ContactForm, 
    FooForm, 
    BarForm,
    )
'''